### ISSUE TITLE
block is empty should allow for field declarations in constructor parameters

### ISSUE THREAD

**Issue by neoGeneva** [palantir/tslint#67]
The following code currently reports as "block is empty", but because it's a constructor with parameters that define fields, the empty block is needed and shouldn't be considered an error.

``` typescript
class Compile implements ng.IDirective {
    constructor(private $compile: ng.ICompileService) {
    }
}
```
